roles/contiv/tasks/delete_contiv.yml

```diff
@@ -0,0 +1,31 @@
+- name: delete contiv
```

Comment:
@srampal playbooks really shouldn't be in tasks directory.  Maybe move this to playbooks/adhoc